src/main/java/kr/co/techmoa/carracing/service/CarMoveService.java

```diff
@@ -0,0 +1,58 @@
+package kr.co.techmoa.carracing.service;
+
+import kr.co.techmoa.carracing.model.Car;
+import kr.co.techmoa.carracing.model.Move;
+import kr.co.techmoa.carracing.model.RacingCarGame;
+
+import java.util.ArrayList;
+
+public class CarMoveService {
+
+    public static final int OPERATION_MAX = 10;
+    public static final int ADVENCE_LIMIT = 4;
+    public static final int FIRST_ROUND = 0;
+
+    public RacingCarGame move(RacingCarGame racingCarGame) {
+
+        int sum = 0;
+
+        ArrayList roundList = new ArrayList<Car[]>();
```

Comment:
[Java 제네릭 - Raw Type을 쓰면 안되는 이유](http://happinessoncode.com/2018/02/08/java-generic-raw-type/)

Reply:
```suggestion
        List<Car[]> roundList = new ArrayList<Car[]>();
```

리스코프 치환 원칙 (Liskov Substitution Principle)
상위 타입의 객체를 하위 타입의 객체로 치환해도 상위 타입을 사용하는 프로그램은 정상적으로 동작해야 한다.